docs/howto/add_pagination_links.md

```diff
@@ -126,14 +126,13 @@ def meta_attributes(resource, extra_meta = {})
   {
     current_page: resource.current_page,
     next_page: resource.next_page,
-    prev_page: resource.prev_page,
+    prev_page: resource.prev_page, # use resource.previous_page when using will_paginate
```

Comment:
Oh, interesting.  Our paginator does all this work for us https://github.com/mislav/will_paginate/blob/2f0b4694b3cc7ffcb265daad13eb95c972fa0bb1/lib/will_paginate/collection.rb#L20